### ISSUE TITLE
SharePoint List items

### ISSUE THREAD

**Issue by JamesMil** [OfficeDev/microsoft-teams-apps-requestateam#123]
SharePoint List items 

Is it possible to customise the SharePoint List items.xlsx with items that our relevant to the company and if not what are the settings for: 

SMB - Company-wide
SMB - Executives Team
SMB - Finance
educationStaff
educationProfessionalLearningCommunity
retailStore
retailManagerCollaboration
healthcareWard
healthcareHospital

**Comment by alexc-MSFT** [comment]
@JamesMil yes, youc an customise the items in the spreadsheet if you wish OR after the deployment by editing the items in the SharePoint lists.

The items listed above are the ids for the standard/out of the box teams templates (these are stored in a list named 'Teams Templates') and can be selected from when you request a team through the app.

**Comment by alexc-MSFT** [comment]
Please let me know if this has answered your question.

**Comment by JamesMil** [comment]
@alexc-MSFT can you set it to retain information such as the Privacy settings and the Owners?

**Comment by JamesMil** [comment]
@alexc-MSFT ran a template and recieved the following:
[
  {
    "message": "Invalid type. Expected String but got Null.",
    "lineNumber": 0,
    "linePosition": 0,
    "path": "value[0].surname",
    "schemaId": "#/properties/value/items/properties/surname",
    "errorType": "type",
    "childErrors": []
  }
]
any advise?
